docs/security/azure-linux-os-guard.md

```diff
@@ -0,0 +1,534 @@
+
+## Objectives
+
+In this lab, you will learn how to:
+- Provision an Azure Linux with OS Guard AKS cluster 
+- Validate Trusted Launch security 
+- Demonstrate immutability enforcement in the /usr directory 
+- Explore advanced SELinux and IPE security policies 
+- Inspect and compare the package count between the Azure Linux Container Host and OS Guard on AKS
+- Perform migration and rollback operations
+
+## Limitations and Considerations
+
+Azure Linux with OS Guard is currently in Public Preview. It's important to be aware of the following limitations and considerations: 
+- Kubernetes version 1.32.0 or higher is required for Azure Linux with OS Guard.
+- All Azure Linux with OS Guard images have [Federal Information Process Standard (FIPS)](/azure/aks/enable-fips-nodes) and [Trusted Launch](/azure/aks/use-trusted-launch) enabled.
+- Azure CLI and ARM templates are the only supported deployment methods for Azure Linux with OS Guard on AKS in preview. PowerShell and Terraform aren't supported.
+- [Arm64](/azure/aks/use-arm64-vms) images aren't supported with Azure Linux with OS Guard on AKS in preview.
+- `NodeImage` and `None` are the only supported [OS Upgrade channels](/azure/aks/auto-upgrade-node-os-image) for Azure Linux with OS Guard on AKS. `Unmanaged` and `SecurityPatch` are incompatible with Azure Linux with OS Guard due to the immutable /usr directory.
+- [Artifact Streaming](/azure/aks/artifact-streaming) isn't supported.
+- [Pod Sandboxing](/azure/aks/use-pod-sandboxing) isn't supported.
+- [Confidential Virtual Machines (CVMs)](/azure/aks/confidential-containers-overview) aren't supported.
+- [Gen 1 virtual machines (VMs)](/azure/aks/aks-virtual-machine-sizes#vm-support-on-aks) aren't supported.
+
+<Prerequisites />
+<ProvisionResourceGroup />
+
+### Scenario 1: Create an Azure Linux with OS Guard on AKS Cluster
+You can either spin up a new cluster or add node pools to an existing cluster to experiment with Azure Linux with OS Guard on AKS. For the purposes of this lab, we will create a new Azure Linux with OS Guard cluster.
+
+::: important
+Azure Linux with OS Guard for AKS is currently in Public Preview, so you will need to register the following preview feature flag using Azure CLI.
+
+```bash
+az feature register --namespace Microsoft.ContainerService --name AzureLinuxOSGuard
+```
+
+It takes a few minutes for the status to show *Registered*. Verify the registration status by using the `az feature show` command:
+
+```bash
+az feature show --namespace "Microsoft.ContainerService" --name "AzureLinuxOSGuardPreview"
+```
+
+When the status reflects *Registered*, refresh the registration of the *Microsoft.ContainerService* resource provider by using the `az provider register` command:
+
+```bash
+az provider register --namespace "Microsoft.ContainerService"
+```
```

Comment:
The user will also need to install the aks-preview az-cli extension to be able to specify `--os-sku AzureLinuxOSGuard`